modules/app/src/main/resources/assets/styles/main-app-wrapper.less

```diff
@@ -11,7 +10,6 @@
     width: @app-header-height;
     position: absolute;
     top: 0;
-    left: 0;
```

Comment:
Any special reason to remove this? `left: 0` is not always implied by default.

Reply:
It is defined again futher down. Duplicate rule "left: -@app-header-height"